index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Technical Documentaion Page</title>
+    <link rel="stylesheet" href="CSS.css">
+</head>
+<body>
+    <nav id="navbar">
+        <header>Programming languages</header>
+        <ul>
+            <li><a href="#JAVA" class="nav-link">JAVA</a></li>
+            <li><a href="#C++" class="nav-link">C++</a></li>
+            <li> <a href="#PYTHON" class="nav-link">PYTHON</a></li>
+            <li> <a href="#JAVASCRIPT" class="nav-link">JAVASCRIPT</a></li>
+            <li><a href="#PHP" class="nav-link">PHP</a></li>
+        
+        </ul>
+    </nav>
+    
+    <main id="main-doc">
+
+        <section class="main-section" id="JAVA"><header>JAVA</header>
+        <h1>JAVA is one of the world most usable programming language,It was introduced by Games Gosling</h1>
+        <p>It is used for the following software</p>
+        <ul> <li>Desktop Application</li> </ul>
+        </section>
+        <br>
```

Comment:
The best practice for professional code review demands that we phrase our comments politely, as suggestions instead of orders. Also, as I suggested in my previous comment, please try to give reasons why you suggest any changes so the author has an opportunity to learn from you. The reason may be obvious to you but not to the author of the code. 

_**Your comments sound like a nice suggestion but I think can do even better.**_ 

For example:
* This is a polite suggestion: *"I think it would be better to use a `div` instead of `section` for semantic reasons and to improve accessibility"*
* This is void of reason: *"I suggest you use dev instead of section."*